### PR TITLE
Vue storyshots

### PR DESCRIPTION
**Pull request by igor-dv**
Issue:  Adding storyshots support for vue

## What I did
Just wanted to check what is the complexity of adding storyshots for vue based on what is already done for angular, and looks like it's easier than angular stuff. 

Probably I need to do some cherry-pick of the main refactoring.

## How to test
For running only the vue sthoryshots:
```
node node_modules\jest\bin\jest.js vueshots
```

### CODE REVIEW

File: addons/storyshots/package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "@storybook/addons": "^3.3.3",
     "babel-core": "^6.26.0 || ^7.0.0-0",
     "react": "*",
-    "react-test-renderer": "*"
+    "react-test-renderer": "*",
+    "vue": "^2.5.13"
```
Comment (Hypnosphi) [inline addons/storyshots/package.json]: I think we should remove the framework peer dependencies instead of adding them

Comment (igor-dv) [reply addons/storyshots/package.json]: Yeah, I think so too.

**Review by ndelangen — APPROVED** [review]

**Comment by igor-dv** [comment]
@ndelangen , @Hypnosphi , WDYT, I'll merge it to the angular branch and continue from there?

**Comment by ndelangen** [comment]
Sounds like a plan @igor-dv absolutely EPIC work!

**Pull request merged**